game-service/src/main/java/org/libertybikes/game/round/service/GameRoundService.java

```diff
@@ -31,6 +32,22 @@
         return allRounds.values();
     }
 
+    @POST
+    @Path("/joinRound/{roundId}")
+    public String joinRound(@PathParam("roundId") String roundId) {
```

Comment:
This rest endpoint is misleading because it doesn't actually cause the player to join a round.  A more accurate name would be `isJoinable`.

However, the sort of validation being done on this method could be done on the front end by calling the existing `GET /round/{roundId}` endpoint and then parsing the returned GameRound JSON object to see what state the round is in and whether or not the round is full.
This would require some updates to make GameRound deserialize into the proper JSON format.